Library/Homebrew/utils/analytics.rb

```diff
@@ -72,7 +72,7 @@ def report_package_event(measurement, package_name:, tap_name:, on_request: fals
         return if not_this_run? || disabled?
 
         # ensure on_request is a boolean
-        on_request = on_request ? true : false
```

Comment:
Some of these corrections may be incorrect, based on the code comment above. We'd need to evaluate the method `sig`s to be sure.

Reply:
`!!` may be the fix here